src/commands/clear.ts

```diff
@@ -15,6 +15,7 @@ module.exports = new Command({
 
 		lobby.pingCleared = true
 		mongoLobbies.updateOne({ _id: lobby._id }, { $set: lobby })
+		mongoSignups.updateMany({ playing: true }, { $set: { playing: false } })
```

Comment:
If we have this now, might be worth specifically removing those users from any voice channel they're in - only person likely to be affected is us as admins?